website/modules/case-studies/index.js

```
@@ -173,4 +240,25 @@ module.exports = {
   },
+  helpers() {
+    return {
+      groupTagsByCategory(tags) {
+        const grouped = {};
+        if (!Array.isArray(tags)) {
+          return grouped;
+        }
+        /* eslint-disable no-underscore-dangle */
+        tags.forEach((tag) => {
+          const category =
+            (tag._category && tag._category[0] && tag._category[0].title) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)